src/backend/xilinx/memory_axi.rs

```diff
@@ -162,13 +165,22 @@ impl MemoryInterface for AxiInterface {
         ));
 
         // bram reading / writing logic
-        bram_logic(&axi4, &mut module, &mode_fsm, "read_txn_count".into());
+        // TODO(nathanielnrn) fix this
```

Comment:
Fix this how? In general TODO comments should attempt to explain what is going on in enough detail that someone can understand it 6 months from now. Either extend the comment or point to an issue